src/components/sections/shareNow/section_share.less

```diff
@@ -0,0 +1,19 @@
+.section_share {
+  margin-bottom: 8rem;
+  .wrapper_shrink {
```

Comment:
You have "wrapper_section-content" inside "section_share". And don't have any "wrapper_shrink". Delete pls